src/org/pentaho/reporting/platform/plugin/async/PentahoAsyncReportExecution.java

```diff
@@ -18,13 +19,13 @@
 
 public class PentahoAsyncReportExecution implements IAsyncReportExecution<InputStream> {
 
-  private final SimpleReportingComponent reportComponent;
-  private final AsyncJobFileStagingHandler handler;
-  private final String url;
-  private final String auditId;
-  private final IPentahoSession safeSession;
+  protected final SimpleReportingComponent reportComponent;
+  protected final AsyncJobFileStagingHandler handler;
+  protected final String url;
+  protected final String auditId;
+  protected final IPentahoSession safeSession;
 
```

Comment:
If you are reusing them there, make that in a separate commit related to that case. Dont wildly mix work on two cases - if you work on two cases at the same time, use separate branches to keep that work cleanly separate.